cmd/container.go

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2019 Ben Overmyer <ben@overmyer.net>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// containerCmd represents the service command
```

Comment:
This comment should refer to the container command, not the service command.

Reply:
Done! You can probably tell what I used for a springboard.